.github/workflows/ci.yaml

```diff
@@ -0,0 +1,101 @@
+name: Publish to ECR
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "helm/**"
+      - "**.md"
+      - "**.txt"
+      - "Jenkinsfile"
+      - ".git**"
+
+  pull_request:
+    paths-ignore:
+      - "helm/**"
+      - "**.md"
+      - "**.txt"
+      - "Jenkinsfile"
+      - ".git**"
+
+jobs:
+  test:
```

Comment:
It seems a bit of a pain that this will mean the project is tested twice (once via Jenkins again by this). It might be better to scope this just to pushing to ECR and not doing testing here?